modules/types/src/dummy_type.cpp

```diff
@@ -0,0 +1,63 @@
+//=================================================================================================
+// Copyright (C) 2023-2024 HEPHAESTUS Contributors
+//=================================================================================================
+
+#include "hephaestus/types/dummy_type.h"
+
+#include <random>
+
+#include "hephaestus/random/random_container.h"
+#include "hephaestus/random/random_type.h"
+#include "hephaestus/types/type_formatting.h"
+
+namespace heph::types {
+
+auto DummyPrimitivesType::random(std::mt19937_64& mt) -> DummyPrimitivesType {
+  return { .dummy_bool = heph::random::randomT<bool>(mt),
+           .dummy_int8_t = heph::random::randomT<int8_t>(mt),
+           .dummy_int16_t = heph::random::randomT<int16_t>(mt),
+           .dummy_int32_t = heph::random::randomT<int32_t>(mt),
+           .dummy_int64_t = heph::random::randomT<int64_t>(mt),
+           .dummy_uint8_t = heph::random::randomT<uint8_t>(mt),
```

Comment:
Add note as comment or in description why only adding fewer types, e.g. bool, uint8,  int16, float32 would be insufficient? In my understanding, you wanna add all primitive types to maximize  test coverage.

Reply:
Can you elaborate, which type are you missing?

Reply:
What I meant was the other way around. Do you want have **all** types as part of your dummy type? If yes -> you could add a comment for the reader why the list of used types shall be as complete as possible.

Reply:
Gotcha. Added a comment, and added both steady clock and system clock to improve coverage.